src/liquid/place/bulk_create_form.html

```diff
@@ -79,6 +79,15 @@
               <td>Property on primary contact</td>
             </tr>
             {% endfor %}
+
+            {% if contactType.user_role.parameter != null %}
+            <tr>
+              <td>{{ contactType.user_role.friendly_name }}</td>
+              <td>{{ contactType.user_role.required }}</td>
+              <td>Role(s) of the user, use when there are multiple roles available at this level of your hierarchy. <br> <i>eg: rcom or supervisor+stock_manager </i> <br> (separate multiple roles with <b>+</b> or leave blank to use the default role of this level.)
```

Comment:
```suggestion
              <td>Role(s) of the user. Separate multiple roles with <b>+</b> or leave blank to use the default role <i>eg: rcom or supervisor+stock_manager </i>
```

I don't think we should say "when there are multiple roles available" because this only shows when there are multiple roles available?